terms.html

```diff
@@ -0,0 +1,86 @@
+  <li><em>Optional account information</em>: Display name, biography, profile information fields, profile picture, and header image. Display name, biography, profile picture and header image will always be public.</li>
+  <li><em>Statuses and interactions</em>: We retain all your posts including attachments, and other interactions (such as favourites, follows and reblogs). In addition to the content and people involved, we also store the timestamps for all of the listed data entries. If these interactions impact another server (eg. following, boosting, or messaging a user on a different server), this other server will receive all required information. Public, unlisted, and pinned posts are available publicly. Follower-only posts are available to your followers, and direct messages are available to you and all people mentioned in the message. Please note that since we cannot control other servers, this means that we cannot guarantee the privacy status of your messages as soon as they leave our server.</li>
+  <li><em>Cookies</em>: We use cookies to keep you logged in and save your preferences for future visits.</li>
+  <li><em>Other metadata</em>: We do not log or store your IP address as a rule. Exceptions will be made when we actively search for errors. Once we have finished the search for errors, collected IP addresses will be deleted. We retain the name of your browser application to allow you to review your currently logged in sessions for security reasons.</li>
+</ul>
+
+<hr class="spacer" />
+
+<h3 id="use">Information usage</h3>
+
+<p>Any of the information we collect from you may be used in the following ways:</p>
+
+<ul>
+  <li>To provide the core functionality of Mastodon. You can only interact with other people's content and post your own content when you are logged in. For example, you may follow other people to view their combined posts in your own personalized home timeline.</li>
+  <li>To aid moderation of the community – when a status or account is reported, we will look into the matter as part of our moderation tasks.</li>
+  <li>The email address you provide may be used to send you information, notifications about other people interacting with your content or sending you messages, and to respond to inquiries, and/or other requests or questions.</li>
+</ul>
+
+<hr class="spacer" />
+
+<h3 id="protect">Information protection</h3>
+
+<p>We implement a variety of security measures to maintain the safety of your personal information when you enter, submit, or access your personal information. Among other things, your browser session, as well as the traffic between your applications and the API, are secured with SSL, and your password is hashed using a strong one-way algorithm. You may enable two-factor authentication to further secure access to your account.</p>
+
+<hr class="spacer" />
+
+<h3 id="data-retention">Information deletion and retention</h3>
```

Comment:
I'd replace SSL with TLS

Reply:
I guess we can actually say HTTPS, because that's clearer.

Reply:
That sounds even better, yes.